mscore/pianoview.cpp

```diff
@@ -532,7 +536,11 @@ void PianoView::wheelEvent(QWheelEvent* event)
             updateNotes();
 
             int mousePixY = (int)(mouseYNote * _noteHeight);
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 1)) //???
```

Comment:
No documentation found since when this replacement exists, it does not in 5.9, and docs are for 1.14.1